packages/playwright-core/src/utils/registry.ts

```diff
@@ -524,22 +524,23 @@ export class Registry {
     const executables = this._addRequirementsAndDedupe(executablesToInstall);
     await fs.promises.mkdir(registryDirectory, { recursive: true });
     const lockfilePath = path.join(registryDirectory, '__dirlock');
-    const releaseLock = await lockfile.lock(registryDirectory, {
```

Comment:
Why does it have to move?

Reply:
that's so that it's inside a try-catch and thus catches the lockfile error